releasenotes/notes/honor-DD_CONTAINER_EXCLUDE_LOGS-f42c6b2fa8aa6b32.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    The agent was collecting docker container logs even if they are matching `DD_CONTAINER_EXCLUDE_LOGS`
```

Comment:
should we mention `DD_CONTAINER_EXCLUDE_METRICS` as well?

Reply:
Indeed. I’ve added `DD_CONTAINER_EXCLUDE_METRICS`.